internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -38,6 +38,8 @@ func NewServer(c config.Configuration) (*Server, error) {
 		return nil, errors.Wrap(err, "Ping Failed")
 	}
 
+	log.WithFields(log.Fields{}).Info("Server was Created")
```

Comment:
```suggestion
	log.Info("Server was Created")
```
No need to use `WithFields` with no fields